modin/core/execution/ray/generic/io/io.py

```diff
@@ -13,250 +13,8 @@
 
 """The module holds base class implementing required I/O over Ray."""
 
-import io
-import os
-import pandas
-
 from modin.core.io import BaseIO
-from modin.core.execution.ray.common.task_wrapper import SignalActor
-import ray
 
 
 class RayIO(BaseIO):
     """Base class for doing I/O operations over Ray."""
-
```

Comment:
the `ray/generic/io.py` became [completely empty](https://github.com/anmyachev/modin/blob/6dfe2695da0d9098bd6bd77e38ae5eb79380b24b/modin/core/execution/ray/generic/io/io.py#L14-L20). Do we depend on `RayIO` class somewhere in our flow? If not, maybe we can remove `io` folder from the `ray/generic` since it doesn't define anything?

Reply:
I would leave it for now, there is a hope that it will be possible to bring the common from the various storage formats to this class. We will have time to delete it yet :)

Reply:
Maybe it is worth removing this class and adding when necessary?

Reply:
Thus, the file too

Reply:
I would not like it since this class reflects the structure we are striving for.